app/labor/rate_limit_checker.rb

```diff
@@ -49,6 +55,11 @@ def check_image_upload_limit
       SiteConfig.rate_limit_image_upload
   end
 
+  def check_article_update_limit
```

Comment:
@Zhao-Andy where are we calling this method `check_article_update_limit ` from?

Reply:
Ah hm I had a git issue so I might not have added... Good catch!

Reply:
Ah never mind, this is called by `limit_by_action` in the same file:
https://github.com/thepracticaldev/dev.to/blob/578d09a54e4554031932c3cf60bf4b390cc5fb50/app/labor/rate_limit_checker.rb#L12-L13